doc/man/wesnoth.6

```diff
@@ -253,24 +253,34 @@ Example:
 For details regarding the preprocessor visit:
 https://wiki.wesnoth.org/PreprocessorRef#Command-line_preprocessor.
 .TP
+.BI --preprocess-string \ source-string
+preprocesses a given string and writes the output to stdout. The common macros
+from the "data/core/macros" directory will be preprocessed before the specified string.
```

Comment:
This is now obsolete.

Reply:
Details? What exactly is obsolete, the manpage or something else?

Reply:
removed.